sdk/dotnet/Pulumi.Automation/ProjectBackend.cs

```diff
@@ -8,5 +11,30 @@ namespace Pulumi.Automation
     public class ProjectBackend
     {
         public string? Url { get; set; }
+
+        internal static IEqualityComparer<ProjectBackend> Comparer { get; } = new ProjectBackendComparer();
```

Comment:
style nitpick - Putting private class definitions at the bottom is good practice I think, but I think `static/const` properties should be defined at the top of a class, before any instance properties